src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -152,6 +103,7 @@ public async Task<(Error, IEnumerable<GitHubUser>)> GetUsersByCommit(string owne
               user {
                 databaseId
                 name
+                email
```

Comment:
do we cover this change in our test?

Reply:
Added a test case where `author.email` is different from `author.user.email`